requirements.txt

```diff
@@ -9,3 +9,4 @@ browser-cookie3
 rsa
 aioredis
 click
+qrcode[pil]
```

Comment:
needs to be in dev requirements and setup.py as well

Reply:
Which file in the dev requirements?

Reply:
Wait nevermind, this isn't gonna be tested